src/server/pkg/backoff/exponential.go

```diff
@@ -114,8 +127,24 @@ func NewInfiniteBackOff() *ExponentialBackOff {
 	} else if b.RandomizationFactor > 1 {
 		b.RandomizationFactor = 1
 	}
-	b.Reset()
-	return b
+	return b.withCanonicalRandomizationFactor().withReset()
+}
+
+// New10sBackoff returns a backoff that's slightly more aggressive than
+// NewExponentialBackOff. The Max Elapsed time for this backoff is 10s, and the
+// initial backoff is 100ms (instead of 500). Therefore this will retry at most
+// 11 times and then fail (depending on RPC timeout), and may be more useful
+// for interactive RPCs than the default timeout of 60s.
+func New10sBackoff() *ExponentialBackOff {
```

Comment:
I adjusted InitialInterval too, since IMO 500ms backoff is also too slow for interactive RPCs.

I can move this out if you want, but there are a bunch of places where we don't retry and should, and I think we should be using this in most of them. I think I'll want to add this back at some point, if not in this PR.

If the name seems bad, I'd be happy to call it something else? NewInteractiveBackoff or something?